ansible/roles/jitsi-videobridge/files/configure-jvb-shards-wrapper.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+[ -z "$SHARDS_LOGDIR" ] && SHARDS_LOGDIR="/var/log/jitsi/jvb-shards"
+[ -z "$TEMPLATE_LOGFILE" ] && TEMPLATE_LOGFILE="$TEMPLATE_LOGDIR/jvb-reconfigure.log"
+
+if [ ! -d "$TEMPLATE_LOGDIR" ]; then
+  mkdir $TEMPLATE_LOGDIR
+fi
+
+if [ ! -f "$TEMPLATE_LOGFILE" ]; then
+  touch $TEMPLATE_LOGFILE
+fi
+
+cp /etc/jitsi/videobridge/shards.json /var/log/jitsi/jvb-shards/$(date --utc +%Y-%m-%d_%H:%M:%S.Z)-shards.json
+
+echo "$(date --utc +%Y-%m-%d_%H:%M:%S.Z) starting configure-jvb-shards.sh" >> $TEMPLATE_LOGFILE
+/usr/local/bin/configure-jvb-shards.sh
+RET=$?
+
+if [ $? -gt 0 ]; then
+    echo "$(date --utc +%Y-%m-%d_%H:%M:%S.Z) jvb shards update failed" >> $TEMPLATE_LOGFILE
+    echo -n "jitsi.jvb.shards_update_update_failed:1|c" | nc -4u -w1 localhost 8125
```

Comment:
was thinking of not having this be in the jitsi.jvb namespace since the JVB itself uses that.  But I could go either way.